### DISCUSSION TITLE
Daily 15/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#276] [Daily Sprint]
- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by Brian-Almada** [comment]
Para hoy voy a terminar con las tareas que tengo en progreso y avanzar con los estudios de los apuntes del curso.

**Comment by NataliaAlvarezIspc** [comment]
El viernes estuve tratando de crear el servicio para login, regristración y restablecer, tuve muchas dificultades. El sábado a la mañana, gracias a la ayuda de un compañero del equipo, entendí como hacerlo. Así que el sábado durante el día estuve agregando los métodos que me faltaban e inyectándolos en los ts correspondientes.  Luego me dediqué a entender cómo se hacían las validaciones para los formularios. Por la noche tuvimos nuestra reunión semanal, en la que vimos como estabamos con las tareas y cosas pendientes nos quedaban por hacer.
El domingo por la noche cree todas las validaciones en los formularios correspondientes y las subí al repositorio.
Hoy por la mañana solo tuve tiempo para crear la daily y las tareas de django. Luego me tuve que ocupar de otros temas y se me complicó la tarde.
Ahora voy a estar pasando los apuntes de la reunión semanal y leyedo los contenidos de la plataforma del ISPC.

**Comment by BraianTroncoso** [comment]
Hoy tengo clases de Python, tengo que recuperar una clase de Java que me quedó pendiente de la semana pasada que voy hacerla ahora, a demás tengo una clase de Game design(son 5, pero ya cursé una diplomatura sobre ello el año pasado asi que solamente va ser conectarme ver que onda y responder los cuestionarios).  El Sáb, estuve en la charla semanal con el grupo y el Dom decidí tomarmelo para descansar un poco. Y finalmente hoy voy a ponerme con issues que pedí que me asignen y ver que tareas puedo realizar.

**Comment by rpgrca** [comment]
Acomodé algo el código e hice que el título se vea igual en todas las páginas. Voy a seguir haciendo pequeños arreglos, consolidar que los límites que tenemos para los nombres de usuario y cosas así sean iguales a los que tenemos en las clases de Python y en los campos de base de datos.
